VirtoCommerce.Storefront/Domain/Security/GraphQlSecurityHelper.cs

```diff
@@ -0,0 +1,89 @@
+namespace VirtoCommerce.Storefront.Domain.Security
+{
+    public static class GraphQlSecurityHelper
+    {
+        public static readonly string SecurityResultFields = $"succeeded errors";
+        public static readonly string AllRoleFields = $"id name permissions";
+        public static readonly string AllExternalLoginFields = $"";
+        public static readonly string AllUserFields = $"id storeId userName password phoneNumber phoneNumberConfirmed email emailConfirmed twoFactorEnabled" +
+            $"accessFailedCount lockoutEnabled lockoutEndDateUtc isAdministrator userType userState contactId roles {{{AllRoleFields}}} externalLogins {{{AllExternalLoginFields}}}";
+
+        public static string CreateUserRequest(this SecurityGraphQLProvider service, string selectedFields = null)
+        => $@"mutation ($command: InputCreateUserType!)
+        {{
+          securityResult: createUser(command: $command)
+          {{
+            { selectedFields ?? SecurityResultFields }
+          }}
+        }}";
+
+        public static string UpdateUserRequest(this SecurityGraphQLProvider service, string selectedFields = null)
+        => $@"mutation ($command: InputUpdateUserType!)
+        {{
+          securityResult: updateUser(command: $command)
+          {{
+            { selectedFields ?? SecurityResultFields }
+          }}
+        }}";
+
+        public static string DeleteUserRequest(this SecurityGraphQLProvider service)
+        => $@"mutation ($command: InputDeleteUserType!)
+        {{
+            deleteContact(command: $command)
```

Comment:
add alias

Reply:
![image](https://user-images.githubusercontent.com/8361000/87168488-dd4bb800-c2ce-11ea-8821-41aa170275fb.png)

Reply:
yes, add the alias to graphql query and make sure to reuse existing SecurityResultResponseDto